lib/js/components/Dropdown/Dropdown.vue

```diff
@@ -9,22 +9,24 @@
 		:trigger="triggerAction"
 		:delay-on-mouse-out="300"
 		@document-click="$emit('document-click')"
-		@hide="$emit('hide')"
-		@show="$emit('show')"
+		@hide="onHide"
+		@show="onShow"
 	>
 		<div
 			class="popper dsDropdown"
 			:class="{
 				'-radiusBottom': radius === DROPDOWN_RADIUSES.BOTTOM,
 				'-radiusTop': radius === DROPDOWN_RADIUSES.TOP,
 				'-radiusBottom -radiusTop': radius === DROPDOWN_RADIUSES.BOTH,
+				'-heightLimited': maxHeight,
 			}"
+			:style="maxHeight ? { maxHeight } : null"
 		>
 			<slot :close="close" />
 		</div>
 
 		<template #reference>
-			<slot name="reference" />
+			<slot name="reference" :is-opened="forceShow ? true : isOpened" />
```

Comment:
I'm not sure about this one:

https://github.com/bethinkpl/design-system/assets/7030884/677a9769-5640-48a5-855b-3a6e03c08e77

Reply:
Did you pull fresh changes?

Reply:
nice, i see now, i think it its wrong implementation in storybook, should we allow to call close when force-show is true?

Reply:
Fixed!